src/cacheDisplay.html

```diff
@@ -94,114 +92,110 @@
-                            <td md-cell>{{item.tag}}</td>
-                            <td md-cell>{{item.index}}</td>
-                            <td md-cell>{{item.offset}}</td>
-                          </tr>
-                        </tbody>
-                    </table>
-                </md-table-container>
+            <!-- Memory Access Queue -->
+            <md-content layout="row" layout-align="space-around center">
+                <md-subheader class="md-no-sticky">{{$ctrl.fileName}}</md-subheader>
+                <md-button class="md-primary md-raised" ng-click="$ctrl.handleUpload()" ng-disabled="$ctrl.fileName.length != 0" type="submit" style="margin-right: 0;">Upload</md-button>
+            </md-content>
+            <md-subheader class="md-no-sticky">Memory Access Queue</md-subheader>
+            <md-table-container style="height:150px">
+                <table md-table>
+                    <thead md-head>
+                      <tr md-row>
+                        <th md-column md-numeric>Address</th>
+                        <th md-column md-numeric>Tag</th>
+                        <th md-column md-numeric>Index</th>
+                        <th md-column md-numeric>Offset</th>
+                      </tr>
+                    </thead>
+                    <tbody md-body>
+                      <tr md-row ng-repeat="item in $ctrl.memQueue">
+                        <td md-cell>{{item.address}}</td>
+                        <td md-cell>{{item.tag}}</td>
+                        <td md-cell>{{item.index}}</td>
+                        <td md-cell>{{item.offset}}</td>
+                      </tr>
+                    </tbody>
+                </table>
+            </md-table-container>
 
-                <!-- Control Flow Buttons -->
-                <md-content layout="row" layout-align="space-around center">
-                    <md-button class="md-icon-button"><i class="material-icons">skip_previous</i></md-button>
-                    <md-button class="md-icon-button"><i class="material-icons">pause</i></md-button>
-                    <md-button class="md-icon-button"><i class="material-icons">play_arrow</i></md-button>
-                    <md-button class="md-icon-button"><i class="material-icons">skip_next</i></md-button>
-                    <md-button class="md-icon-button"><i class="material-icons">replay</i></md-button>
-                </md-content>
-                <md-content layout="row" layout-align="center" style="padding-top: 25px; padding-right: 10px;">
-                    <div layout layout-align="center center">
-                        <span class="md-body-1">Speed</span>
-                    </div>
-                    <md-slider flex class="md-primary" md-discrete ng-model="$ctrl.speedRating" step="1" min="1" max="5" aria-label="rating">
-                    </md-slider>
-                </md-content>
+            <!-- Control Flow Buttons -->
```

Comment:
what were all these changes? was it just white spacing?

Reply:
Yeah everything is mostly the same except for the section tag at the top, this was all previously embedded in a body tag that I removed so it changed the indentation for everything